src/lib.rs

```diff
@@ -93,9 +93,11 @@ mod init;
 #[cfg(any(feature = "tokio_lib", feature = "async_std_lib"))]
 pub type PagerMutex = Mutex<Pager>;
 
+pub type ExitCallbacks = Vec<Box<dyn FnMut() + Send + Sync + 'static>>;
+
 /// A struct containing basic configurations for the pager. This is used by
 /// all initializing functions
-#[derive(Clone)]
+// #[derive(Clone)]
```

Comment:
I had to remove clone, as the on_exit_callbacks were not compatible with clone (I believe).

I don't think clone was ever used on this struct anyway, just on the surrounding Arc?